Library/Homebrew/cmd/prune.rb

```diff
@@ -47,5 +48,7 @@ def prune
       print "and #{d} directories " if d > 0
       puts "from #{HOMEBREW_PREFIX}"
     end unless ARGV.dry_run?
+
+    unlinkapps_prune(:dry_run => ARGV.dry_run?, :quiet => true)
```

Comment:
While you're here I wonder if it's worth just adding to `uninstall` too? No worries if you'd rather leave it for another PR.